sdk/src/services/BalanceSubscriptionsService.ts

```diff
@@ -0,0 +1,263 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+import type { CancelablePromise } from '../core/CancelablePromise';
+import type { BaseHttpRequest } from '../core/BaseHttpRequest';
+
+export class BalanceSubscriptionsService {
+
+    constructor(public readonly httpRequest: BaseHttpRequest) {}
+
+    /**
+     * Get balance subscriptions
+     * Get all balance subscriptions.
+     * @returns any Default Response
+     * @throws ApiError
+     */
+    public getAllBalanceSubscriptions(): CancelablePromise<{
+        result: Array<{
+            id: string;
+            /**
+             * A chain ID ("137") or slug ("polygon-amoy-testnet"). Chain ID is preferred.
+             */
+            chain: string;
+            /**
+             * A contract or wallet address
+             */
+            contractAddress?: string;
+            /**
+             * A contract or wallet address
+             */
+            walletAddress: string;
```

Comment:
why separate walletAddress / contractAddress? what's the diff?

in my mind i should just be able to watch any address, doesnt matter if its a contract or not

Reply:
Contract is for erc20 balance